include/mega/sync.h

```diff
@@ -26,31 +26,44 @@
 #include "megaclient.h"
 
 namespace mega {
+
```

Comment:
The following four functions are free, pure functions to make the functionality they implement easier to unit test.